test/service/test_response.py

```diff
@@ -848,6 +848,8 @@ def test_projects_key_search_response(self):
                             "projectTitle": "Single cell transcriptome patterns.",
                             "publications": [
                                 {
+                                    "doi": "10.1016/j.cell.2017.09.004",
```

Comment:
Can we augment the can used by this test to also have the other fields?

Reply:
The property `official_hca_publication` was added in https://schema.humancellatlas.org/module/project/7.0.0/publication , which requires https://schema.humancellatlas.org/type/project/15.0.0/project or higher.

Instead of augmenting the canned bundle mentioned (which uses https://schema.humancellatlas.org/type/project/9.0.3/project), I've taken `test_project_contact_extraction` from test_hca_indexer.py (which asserts project contact fields) and moved / transformed it into `test_projects_response` in test_response.py to use an existing canned bundle there that already has the desired properties.